modules/TargetSafety.py

```diff
@@ -79,11 +85,31 @@ def main(
         .agg(
             F.collect_set(F.col("biosample")).alias("biosamples"),
             F.collect_set(F.col("study")).alias("studies"),
+            F.collect_set(F.col("supporting_variation")).alias("supporting_variation"),
```

Comment:
I think variation is a collective noun, so it can refer to multiple.

Reply:
In terms of the model, you're absolutely right and it was designed like this because it wasn't a problem _yet_.

Effectively speaking, there are 3 sources using this field:
- ToxCast. A study represents an assay, and we never have information about the tissue. So studies are not null, biosample is null. No dependency issues
- Curation of Force et al. A study represents a detailed description of the liability. Studies are not null but is always one, biosample is not null. Dependency is not lost
- PharmGKB. A study represents every drug for which the adverse reaction is observed. Studies are not null, biosample is null.

Even though we don't have problems, we could very well incur in dependency issues. Because of the implication of remodelling this, I'd suggest leaving it like this until we see an issue. What do you think?